magenta/interfaces/midi/midi.py

```diff
@@ -247,16 +264,25 @@ class MonoMidiPlayer(threading.Thread):
     _outport: The Mido port for sending messages.
     _sequence: The monohponic, chronologically sorted NoteSequence to play.
     _stop_playback: A boolean specifying whether the playback should stop.
+    _bpm: The tempo in beats per minute. Copied from input sequence.
   Args:
     outport: The Mido port for sending messages.
     sequence: The monohponic, chronologically sorted NoteSequence to play.
```

Comment:
s/monohponic/monophonic
Incidentally I don't think monophonic is really the right word. Single voice? But it gets the point across.